tests/integration/test_parse_issues.py

```diff
@@ -69,3 +69,20 @@ def test_parse_labels(issue_list, process_issues):
         issue.labels = labels
         review = process_issues.parse_issue(issue)
         assert review.labels == ["6/pyOS-approved", "another_label"]
+        assert review.active
+
+    # Now add an archive label
+    label_inst = Labels(
+        id=1196238794,
+        node_id="MDU6TGFiZWwxMTk2MjM4Nzk0",
```

Comment:
Codecov is noticing some test gaps - can you please ensure that we have full coverage via this pr?

Reply:
Full coverage achieved ✅